.github/workflows/build.yml

```diff
@@ -10,11 +11,11 @@ jobs:
         nixCommand:
```

Comment:
```suggestion
```

Reply:
I'd like to keep that here, because we will add more commands soon, like `nix build .#holochain-tests-nextest` etc.